evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -113,7 +113,7 @@ contract FluxAggregator is AggregatorInterface, Owned {
     address newAdmin
   );
   event SubmissionReceived(
-    int256 indexed answer,
+    int256 indexed response,
```

Comment:
Should this be something like "value" ? I'm confused by response in this context, where is the request?

Reply:
Just trying to make it consistent.

Reply:
"Value" is a little overloaded and vague. Went with "submission," the whole tuple distinction felt a bit like hair splitting; it's a submission for a round.